### ISSUE TITLE
Keyboard Shortcuts

### ISSUE THREAD

**Issue by lukewestby** [HumbleSpark/ellie-issues#2]
This is already on the roadmap but I want to take suggestions about what combos there should be. If you have a suggestion, please reference another tool that uses that particular shortcut/command combo if you know of one or some thoughts about why the command is the best user experience. Totally cool if you don't want to do this but I will have to think through more whether suggestions without citations will be good for users.

Things we've got so far:
- <kbd>cmd/ctrl</kbd>-<kbd>enter</kbd>: run compiler

Thanks!

**Comment by BrianHicks** [comment]
My top and only item: <kbd>cmd/ctrl</kbd>-<kbd>enter</kbd> to compile

**Comment by listrophy** [comment]
Yeah, I honestly only care about compilation right now. If this weren't a browser app, I'd prefer `cmd/ctrl-r`, but please don't actually do that. 😀  I think I can get behind `cmd/ctrl-enter`.

**Comment by lukewestby** [comment]
Where does <kbd>cmd/ctrl</kbd>-<kbd>enter</kbd> come from?

**Comment by BrianHicks** [comment]
It's been the "run/compile" shortcut in most online editors that I've used, and more than a few IDEs. <kbd>ctrl</kbd>-<kbd>enter</kbd> is used in the "try online" portion of elm-lang.org. I'd like it to also be <kbd>cmd</kbd>-<kbd>enter</kbd> because, well, I use a mac. ;)

**Comment by lukewestby** [comment]
Excellent, thanks! Done deal.

**Comment by basile-henry** [comment]
<kbd>cmd/ctrl</kbd>-<kbd>shift</kbd>-<kbd>f</kbd> for format.

Alternatively, and probably a better solution, would be to have an option to format on compile.

**Comment by allanderek** [comment]
I would love a keyboard shortcut for indenting/unindenting a block of code. I'm not too opinionated on what that keyboard shortcut should be, I think Tab (and Shift+Tab) are common, but as I say I'm none too fussed myself.

**Comment by andys8** [comment]
Need this! @lukewestby  :D
